site/content/docs/support.md

```diff
@@ -7,7 +7,7 @@ The table below provides a matrix of supported .NET application types and AWS Co
 | Amazon Elastic Container Service (ECS) service| X             | X                 |           |
 | Amazon Elastic Container Service (ECS) task	| X             | X                 |           |
 | AWS App Runner    |              | X                 |           |
-| AWS Elastic Beanstalk     |               | X                 |           |
+| AWS Elastic Beanstalk (Linux and Windows)     |               | X                 |           |
```

Comment:
If we are specifying the platform in the description for the AWS Elastic Beanstalk, we should specify it for other computes as well. I suggest we do it for each one, e.g AWS App Runner (Linux), AWS Elastic Beanstalk (Linux and Windows), etc...

Reply:
Done